disnake/message.py

```diff
@@ -800,6 +800,10 @@ class Message(Hashable):
         .. versionadded:: 2.0
     guild: Optional[:class:`Guild`]
         The guild that the message belongs to, if applicable.
+    thread: Optional[:class:`Thread`]
+        The thread that was started from this message, if applicable.
+
+        ..versionadded:: 2.4
```

Comment:
This isn't needed anymore, as it now uses the documentation of the property.

Reply:
Some day I will learn how sphinx works